patches.json

```
@@ -643,5 +643,14 @@
         }
       }
     }
+  },
+  "MC-38509": {
+    "magento/magento2-base": {
+      "Fixes the issue where the \"Create an Account\" button left disabled even after correcting invalid create account form.": {
+        ">=2.3.0 <2.3.6-p1 || >=2.4.0 <2.4.2": {
```

The fix will be delivered in 2.4.1-p1
```suggestion
        ">=2.3.0 <2.3.6-p1 || >=2.4.0 <2.4.1-p1": {
```